xml/System.Diagnostics.Tracing/EventSource.xml

```diff
@@ -241,7 +241,16 @@
       <Docs>
         <param name="eventSourceName">The name to apply to the event source. Must not be <see langword="null" />.</param>
         <summary>Creates a new instance of the <see cref="T:System.Diagnostics.Tracing.EventSource" /> class with the specified name.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+  ## Remarks
+
+  When using this constructor, ensure that the `eventSourceName` argument matches the ETW name defined by the <xref:System.Diagnostics.Tracing.EventSourceAttribute> attribute on that type. Otherwise, the GUIDs returned by the <xref:System.Diagnostics.Tracing.EventSource.Guid> property and the <xref:System.Diagnostics.Tracing.EventSource.GetGuid(System.Type)> method will be different.
```

Comment:
```suggestion
  When using this constructor, ensure that the `eventSourceName` argument matches the ETW name defined by the <xref:System.Diagnostics.Tracing.EventSourceAttribute> attribute on that type. Otherwise, the GUIDs returned by the <xref:System.Diagnostics.Tracing.EventSource.Guid> property and the <xref:System.Diagnostics.Tracing.EventSource.GetGuid(System.Type)> method will be different. If the event source names differ, the <xref:System.Diagnostics.Tracing.EventSource.Guid> property returns the GUID used to register this EventSource with ETW.
```